### ISSUE TITLE
No stereo output

### ISSUE THREAD

**Issue by mrk67422** [8bitbubsy/ft2-clone#45]
Does not play stereo .mod files in stereo.
Does FT2 support stereo output?

**Comment by 8bitbubsy** [comment]
FT2 supports stereo, yes, but it doesn't have stereo separation when it imports .MOD files. This is how it is on original FT2 (MS-DOS) as well.

**Comment by mrk67422** [comment]
Thank you 8bitbubsy for your quick reply. By the way, congratulations on a very successful project, a great and useful job. A long, long time ago I was the owner of an A500. A friend of mine recently brought up the topic of .mod files and the wonderful period of life when you created whole collections filled with these files. As a sentimental person, I got into the topic, but I bounced off the wall right away. The topic of Trackers is as wide as the world. So I started trying different applications, started listening to mods and got lost. I’ll start by saying that I didn’t remember how stereo was organized in the A500. Now I know that channels 1 and 4 of the sound system Paula were responsible for Stereo Left, and channels 2 and 3 for Stereo Right. And it was with full separation. As I have a good and sensitive hearing, I started to go crazy, because every program generated a different stereo background. Some sounded terribly mono to me, others hit the ear with one instrument, which was unpleasant. 8bitbubsy led me to the concept of Panning. I already know that hard Panning Amiga 500 means that some tracks will sound bad if we play them on well-separated stereo speakers or headphones. The point is that Amiga had too few channels to organize good stereo in this way and it may happen that the bass drum will hit us only in one ear which is unpleasant. The solution is Panning L R, where we set the percentage of penetration of one to the other. We make the sound more mono, but with the right proportions it sounds better than with hard separation. So, 8bitbubs, your answer already leads me. If someone  want to have an authentic Fastracker, so .mod from Amiga will be only mono. I’ll give a hint. If we want to have stereo, just use Protraker 2.3d Clone, also produced by 8bitbubsy. There we can control the proportions of separation using Shift-F12 or in the configuration file. MilkyTrucker is also very cool, but there you will encounter hard separation and you don’t have much influence on it. In MilkyTracker you will also have problems with resolutions and scaling, but I’ll tell you that in Windows in compatibility settings, set high resolution DPI, replacing scaling behavior and only then in the MilkyTacker configuration file modify resolutions and scaling. A professional tool is also OpenMPT, but this is already contemporary software. No matter how we look at it, it’s best to go back to the classics, and experience it as it was when the music of 16-bit computers changed the world. That’s why I recommend Fastracker II Cloene and ProTracker 2 Clone by 8bitbubsy. I apologize to you 8bitbubsy for this off top, but since the topic has already started here, maybe someone will get here someday and save a lot of time in this maze of information.
